android/src/main/java/com/thebluealliance/androidclient/adapters/MatchListAdapter.java

```diff
@@ -34,6 +34,11 @@ public void setTeamKey(String teamKey) {
 
     @Override
     public View getChildView(int groupPosition, int childPosition, boolean isLastChild, View convertView, ViewGroup parent) {
+        if (groupPosition >= mGroups.size()
```

Comment:
It's good that we do a bounds check, but it's weird that we'd get into this situation in the first place. Is this worth looking deeper into?